### PR TITLE
[MIG][ADD][REM][IMP] medical_medicament: v10 and improvements

### PR DESCRIPTION
**Pull request by obulkin**
* Migrate code to v10
* Change copyright to LGPL
* Add notion of active and inactive medicament components
* Remove medicament storage instructions, which are now handled in a more reusable way in ``medical_medicament_storage``
* Make various other improvements to medicament model
* Update unit tests, views, menu items, manifest, and README accordingly
* Clean up code, bringing it more in line with OCA guidelines

### CODE REVIEW

File: medical_medicament/models/medical_medicament.py [hunk 1]
```diff
@@ -1,110 +1,167 @@
...
+            ('a', 'A'),
+            ('b', 'B'),
+            ('c', 'C'),
+            ('d', 'D'),
+            ('x', 'X'),
+            ('n', 'N'),
+        ],
+        help=(
+            '** FDA Pregancy Categories ***\n'
+            'CATEGORY A :Adequate and well-controlled human studies have'
```
Comment (lasley) [inline medical_medicament/models/medical_medicament.py]: Fix the `:`s please

File: medical_medicament/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    is_medicament = fields.Boolean(
+        readonly=True,
+        help='Checked if the product is a medicament',
+    )
+    is_vaccine = fields.Boolean(
```
Comment (lasley) [inline medical_medicament/models/product_template.py]: This isn't implemented anywhere - go ahead and remove

**Review by lasley — APPROVED** [review]
Thanks @obulkin - few tiny comments but not in functional parts

**Comment by obulkin** [comment]
I've pushed up a commit that addresses these comments.

**Pull request closed without merging**